webapp/src/components/post_type_jitsi/post_type_jitsi.tsx

```diff
@@ -97,14 +96,13 @@ export class PostTypeJitsi extends React.PureComponent<Props, State> {
 
         const props = post.props;
 
-        let meetingLink = props.meeting_link;
+        let meetingLink = encodeURIComponent(props.meeting_link);
         if (props.jwt_meeting) {
-            meetingLink += '?jwt=' + (props.meeting_jwt);
+            meetingLink += encodeURIComponent('?jwt=' + (props.meeting_jwt));
         }
 
-        meetingLink += `#config.callDisplayName="${props.meeting_topic || props.default_meeting_topic}"`;
-        meetingLink += `&userInfo.displayName="${this.props.currentUser.username}"`;
-        meetingLink = encodeURI(meetingLink);
+        meetingLink += encodeURIComponent(`#config.callDisplayName="${props.meeting_topic || props.default_meeting_topic}"`);
+        meetingLink += encodeURIComponent(`&userInfo.displayName="${this.props.currentUser.username}"`);
```

Comment:
I'm not sure if you're supposed to url encode the `&` character like this, though not sure. Have you made sure this works properly? Can you post the link that's created here?

Reply:
@mickmister I read about it and you are right. I have made the necessary changes and now the URL is working properly. 
```
https://meet.jit.si/testing-nukierfwpqmhnxcykiea#config.callDisplayName=%22testing%22&userInfo.displayName=%22kshitijkat%22
```

Reply:
We really should be utilizing `new URLSearchParams`, but we have to do things like `#` instead of `&` here which is why I didn't originally suggest it https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams